stock/templates/registered/operationmanagement.html

```diff
@@ -23,57 +23,90 @@ <h1 class="page-header text-center">{% trans "Fleet Operation Timers" %}
         </div><div class="label label-info text-left" id="current-time"></div>
         <br  />
      </div>
-     {% if optimer %}
-        <table class="table table-responsive">
-            <tr>
-                <th class="text-center">{% trans "Operation Name" %}</th>
-		<th class="text-center">{% trans "Doctrine" %}</th>
-                <th class="text-center">{% trans "Form Up System" %}</th>
-                <th class="text-center">{% trans "Form Up Location" %}</th>
-                <th class="text-center">{% trans "Start Time" %}</th>
-                <th class="text-center">{% trans "Local Time" %}</th>
-		<th class="text-center">{% trans "Duration" %}</th>
-                <th class="text-center">{% trans "FC" %}</th>
-                <th class="text-center">{% trans "Details" %}</th>
-		<th class="text-center">{% trans "Post Time" %}</th>
-		{% if perms.auth.optimer_management %}
-                	<th class="text-center">{% trans "Creator" %}</th>
-		        <th class="text-center">{% trans "Action" %}</th>
-                {% endif %}
-            </tr>
+     <h4><b>{% trans "Next Timers" %}</b></h4>
+     {% if future_timers %}
+     <table class="table">
+        <tr>
+            <th style="width:150px" class="text-center">{% trans "Operation Name" %}</th>
```

Comment:
Fixed widths are not very bootstrap-y. Use the Bootstrap [grid](https://getbootstrap.com/css/#grid) if you need to.